docs/basic-docs/atomicdex-api-20-dev/get_locked_amount.md

```diff
@@ -0,0 +1,86 @@
+# get\_locked\_amount
+
+The `get_locked_amount` method returns the amount of a coin which is currently locked by a swap which is in progress. If the coin queried is not activated, a `NoSuchCoin` error will be returned.
+
+
+#### Arguments
+
+| Parameter | Type   | Description                                                          |
+| --------- | ------ | -------------------------------------------------------------------- |
+| coin      | string | The ticker of the coin you want to query.                            |
+
+
+#### Response
+
+| Parameter                    | Type            | Description                                                                                                    |
+| ---------------------------- | --------------- | -------------------------------------------------------------------------------------------------------------------- |
+| id                           | integer         | Identifies a response to allow matching it with a request. Defaults to `null` if `id` not provided in request.       |
```

Comment:
`id` is part of [2.0 RPC protocol](https://developers.atomicdex.io/basic-docs/atomicdex-api-20/). It can be removed from here.

Reply:
done